src/hb-set.hh

```diff
@@ -506,16 +506,34 @@ struct hb_set_t
 
   bool is_subset (const hb_set_t *larger_set) const
   {
-    if (get_population () > larger_set->get_population ())
-      return false;
+	if(unlikely(larger_set->page_map.length == 0)) {
+		return false;
```

Comment:
Should be:
`return this->is_empty ();`

Since if this set has no elements then it is a subset of larger_set, which has no elements.

Reply:
Ah nice I didnt see those earlier, updated to use some of the convenience functions.